ansible/deploy.yml

```diff
@@ -117,6 +151,10 @@
         node: "{{ seed_nodes[inventory_hostname] }}"
       tags: always
       when: inventory_hostname in seed_nodes
+    - name: Check inventory for seed nodes
+      ansible.builtin.fail:
+        msg: Seed node not defined in network config
+      when: node is not defined
```

Comment:
Needs rebasing? These pre_task fail steps were removed in the last PR to `feat/dashmate` branch, they shouldn't be re-added here